lc0/src/engine.cc

```diff
@@ -37,6 +37,8 @@ const char* kWeightsStr = "Network weights file path";
 const char* kNnBackendStr = "NN backend to use";
 const char* kNnBackendOptionsStr = "NN backend parameters";
 const char* kSlowMoverStr = "Scale thinking time";
+const char* kMoveOverheadStr = "Move Overhead";
+const char* kMinThinkingTimeStr = "Minimum thinking time";
```

Comment:
If minimum thinking time is useless, maybe remove it and instead fix GetBestMoveInternal()?
(if `(root_node_->GetN() <= 1)` call `GetBestChild` instead of `GetBestChildWithTemperature` even if `temperature` is 0)

Reply:
It's useful in the case where move overhead is high (so there is a buffer of time) and thinking for some amount of time increases your strength enough to keep even/win. I'll remove it because I think it needs more thought and I'll just keep the scope of this PR simple. I'll fix the GetBestMoveInternal thing though